src/class-wpml-elementor-translatable-nodes.php

```diff
@@ -719,6 +719,56 @@ public static function get_nodes_to_translate() {
 					),
 				),
 			),
+            'archive-posts' => array(
+                'conditions' => array( self::TYPE => 'archive-posts' ),
+                'fields'     => array(
+                    array(
+                        'field'       => 'archive_cards_meta_separator',
+                        'type'        => __( 'Cards: Separator Between', 'sitepress' ),
+                        'editor_type' => 'LINE'
+                    ),
+                    array(
+                        'field'       => 'archive_cards_read_more_text',
+                        'type'        => __( 'Cards: Read More Text', 'sitepress' ),
+                        'editor_type' => 'LINE'
+                    ),
+                    array(
+                        'field'       => 'nothing_found_message',
+                        'type'        => __( 'Nothing Found Message', 'sitepress' ),
+                        'editor_type' => 'AREA'
+                    ),
+                    array(
+                        'field'       => 'pagination_prev_label',
+                        'type'        => __( 'Previous Label', 'sitepress' ),
+                        'editor_type' => 'LINE'
+                    ),
+                    array(
+                        'field'       => 'pagination_next_label',
+                        'type'        => __( 'Next Label', 'sitepress' ),
+                        'editor_type' => 'LINE'
+                    ),
+                    array(
+                        'field'       => 'archive_classic_meta_separator',
+                        'type'        => __( 'Classic: Separator Between', 'sitepress' ),
+                        'editor_type' => 'LINE'
+                    ),
+                    array(
+                        'field'       => 'archive_classic_read_more_text',
+                        'type'        => __( 'Classic: Read More Text', 'sitepress' ),
+                        'editor_type' => 'LINE'
+                    ),
+                ),
+            ),
+            'search-form' => array(
+                'conditions' => array( self::TYPE => 'search-form' ),
+                'fields'     => array(
+                    array(
+                        'field'       => 'placeholder',
+                        'type'        => __( 'Placeholder', 'sitepress' ),
+                        'editor_type' => 'LINE'
+                    ),
+                ),
+            ),
```

Comment:
@amElnagdy, there's still something wrong with the indentation. Do you have the same settings as below?

![indentation](https://user-images.githubusercontent.com/1941656/58133436-31429b80-7bfa-11e9-84cd-f99b6b1861d1.png)

Reply:
not really, the Use tab character and smart tabs options were not selected.